src/forms/AdminUserForm.tsx

```diff
@@ -33,10 +33,7 @@ const AdminUserForm = createForm({
     address: v.user?.address ?? '',
     non_giver: v.user?.non_giver ?? false,
     fixed_non_receiver: !!v.user?.fixed_non_receiver ?? false,
-    non_receiver:
-      !!v.user?.fixed_non_receiver ||
-      !!v.user?.non_receiver ||
-      !v.circle.default_opt_in,
+    non_receiver: !!v.user?.fixed_non_receiver || !!v.user?.non_receiver,
```

Comment:
Ok, I've been thinking through this line and I agree that this change makes sense.

If circle.default_opt_in is false, but the user has opted-in, it should show them as "opt-in", and this makes it show as opted-out, which is a bug.

great catch!

Reply:
> This is working when I opt the user out but not when I use the "Force Opted Out" button

![forceOptouttest](https://user-images.githubusercontent.com/15604932/158516785-3c0c80f3-b5b4-4839-a3b2-f29b0afb1cd4.gif)

This is the flow I was testing and works for me. Let me know if you did something different :)